src/iop/clipping.c

```diff
@@ -2347,22 +2347,26 @@ void gui_init(struct dt_iop_module_t *self)
   GtkWidget *page2 = self->widget = GTK_WIDGET(gtk_box_new(GTK_ORIENTATION_VERTICAL, 0));
 
   g->cx = dt_bauhaus_slider_from_params(self, "cx");
+  dt_bauhaus_slider_set_digits(g->cx, 4);
   dt_bauhaus_slider_set_factor(g->cx, 100.0);
   dt_bauhaus_slider_set_format(g->cx, "%0.f %%");
```

Comment:
Don't we need %0.2 here ? Otherwise the values are displayed as plain integer in the interface. Likewise below.

Reply:
In fact we need only "%0.1"

Reply:
Don't _need_ to. There are many places where there are more digits than are visible. I didn't intend to "clean" all of those up, as it may not be a bad thing anyway. As i said, i think we could get rid of rounding to nearest "digit" completely, except for integers.

Reply:
Ok, let's merge this then. I understand that there is probably many cases like this. I think it makes sense to at least fix this one for the release. I'll issue a PR.

Reply:
To be clear, I didn't consider this a bug and that's why I didn't "fix" it. Rather than showing a lot of digits in the format, the user should just look at the image to see if the cropping is correct.

If you _do_ for some reason want to enforce "consistency" between the number of digits and the format, the cases where they don't match are fairly easy to spot in the spreadsheet I attached. The fixes are all trivial. Again, I didn't do that because I didn't consider it a good idea, but it is up to you.

The number of digits itself (not the format) _is_ important, because the slider is rounding to them, so if not enough digits are set, the user is unable (even invisibly) to get more precision. That's why I submitted this PR to fix such regressions.

Reply:
> If you _do_ for some reason want to enforce "consistency" between the number of digits and the format,

That's not about consistency. What I want is also a fix, on the crop&rotate if you ctrl-scroll on any slider in margin tab, you have no feedback about the change in the value. You need to do 10 changes to see the value go from 0% to 1% for example. I consider this a bug. And this is a regression (due to this PR) as previously the scroll or ctrl-scroll was both changing the value by step of 1%, so always having a visual feedback.

Also, if you edit the slider and enter .5 the value displayed is still 0%. Really a bug to me.

Hope this clarify my thinking.